typescript/cli/package.json

```diff
@@ -30,7 +30,7 @@
     "typescript": "^5.1.6"
   },
   "scripts": {
-    "hyperlane": "node ./dist/cli.js",
+    "hyperlane": "NODE_OPTIONS='--experimental-loader ts-node/esm/transpile-only --no-warnings=ExperimentalWarning' node ./dist/cli.js",
```

Comment:
I think only the `--no-warnings=ExperimentalWarning` is relevant here but anyway we won't need this once I'm finished with https://github.com/hyperlane-xyz/issues/issues/917

Reply:
Gotcha– this makes total sense. Will close out PR in favor of https://github.com/hyperlane-xyz/issues/issues/917 :)